mozilla-ai/lumigator
# Experiment logs

## What's changing
In the experiment details sidepanel now there are 2 tabs: 

1. Experiment details (contains all the information of the selected experiment)
2. Logs ( if the selected experiment is in 'running' status logs will automatically update and scroll, if the experiment is finished the latest log entry is displayed) 

## How to test it

1. Start a new experiment
2. Show it on the right panel 
3. Check the logs button next to status

<img width="1445" alt="Screenshot 2024-12-03 at 12 08 34" src="https://github.com/user-attachments/assets/92e38468-2406-4c4c-bb09-ca103a0f80bf">
